sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -437,32 +437,51 @@
                 "description": "Feature store online store config"
             }
         },
-        "offline_store_target": {
```

Comment:
will this be a breaking change to existing cli/SDK user?

Reply:
no, it will not. this interface is just for arm template. I've run the against e2e tests and it doesn't break.